src/OrchardCore.Cms.Web/appsettings.json

```diff
@@ -79,6 +79,35 @@
     //  "LogoffPath": "Users/LogOff",
     //  "ChangePasswordUrl": "ChangePassword",
     //  "ExternalLoginsUrl": "ExternalLogins"
+    //},
+    //"OrchardCore_AutoSetup": {
+    //  "TriggerSetupUrl": "",
+    //  "RootTenant": {
+    //    "SiteName": "AutoSetup Example",
+    //    "SiteTimeZone": "Europe/Amsterdam",
+    //    "AdminUsername": "Admin",
+    //    "AdminEmail": "info@orchardproject.net",
+    //    "AdminPassword": "OrchardCoreRules1!",
+    //    "DatabaseProvider": "Sqlite",
+    //    "DatabaseConnectionString": "",
+    //    "DatabaseTablePrefix": "",
+    //    "RecipeName": "Agency"
+    //  },
+    //  "SubTenants": [
+    //    {
+    //      "SiteName": "AutoSetup Sub Tenant",
+    //      "SiteTimeZone": "Europe/Amsterdam",
+    //      "AdminUsername": "tenantadmin",
+    //      "AdminEmail": "tenant@orchardproject.net",
+    //      "AdminPassword": "OrchardCoreRules1!",
```

Comment:
I don't like the idea of putting the admin password in appsettings. 
It should be by environment variable, and here for demo purposes, at least a comment to that effect

Reply:
Right, I'm using Env. Variables too, I can remove example from appsettings.json and put them to launchSettings.json?

Reply:
Added Warning comment into appsettings.json